### PR TITLE
Feature/add support for vuejs

### PR DESCRIPTION
**Pull request by arapl3y**

### CODE REVIEW

File: scripts/vue2.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+echo
+
+if ! type "node" > /dev/null; then
+    echo "${ERR}Node not found!" 1>&2
+    echo "Please follow the install instructions at https://nodejs.org/en/ then try again${NC}" 1>&2
+    exit
+fi
+
+if ! type "vue" > /dev/null; then
+    echo "${ERR} Vue CLI not found!"
```
Comment (liamfiddler) [inline scripts/vue2.sh]: I'd recommend adding a `${NC}` to the end of this line so only the error is highlighted.

File: scripts/vue2.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+echo "${BLUE}Furbish:${NC} Initialising Vue project using MF template"
+echo
+
+vue init MentallyFriendly/vue2-boilerplate $appname
+cd ./$appname
+
+if ! type yarn > /dev/null; then
+    echo
+    echo "${ERR}Yarn not found!" 1>&2
+    echo "Please follow the install instructions at https://yarnpkg.com/en/docs/install then try again${NC}" 1>&2
```
Comment (liamfiddler) [inline scripts/vue2.sh]: I'd recommend we check Yarn exists _before_ we run `vue init`, that way it doesn't create the directory before Yarn is installed.

File: scripts/vue2.sh (hunk 1 above)
Comment (liamfiddler) [reply scripts/vue2.sh]: [This is how we've done it for Polymer](https://github.com/MentallyFriendly/furbish/blob/master/scripts/polymer3.sh#L24).

Comment (arapl3y) [reply scripts/vue2.sh]: I'll rework it to replicate the polymer example more closely and just install the vue-cli if it's not found.

File: scripts/vue2.sh (hunk 2 above)
Comment (arapl3y) [reply scripts/vue2.sh]: Good point, will rework.

**Pull request merged**